worlds/stardew_valley/test/__init__.py

```diff
@@ -1,75 +1,164 @@
 import os
 import unittest
 from argparse import Namespace
-from typing import Dict, FrozenSet, Tuple, Any, ClassVar
+from typing import Dict, FrozenSet, Tuple, Any, ClassVar, Iterable
 
-from BaseClasses import MultiWorld
+from BaseClasses import MultiWorld, CollectionState
 from Utils import cache_argsless
-from test.TestBase import WorldTestBase
+from test.bases import WorldTestBase
 from test.general import gen_steps, setup_solo_multiworld as setup_base_solo_multiworld
-from .. import StardewValleyWorld
-from ..mods.mod_data import ModNames
 from worlds.AutoWorld import call_all
+from .. import StardewValleyWorld
+from ..mods.mod_data import all_mods
 from ..options import Cropsanity, SkillProgression, SpecialOrderLocations, Friendsanity, NumberOfLuckBuffs, SeasonRandomization, ToolProgression, \
-    ElevatorProgression, Museumsanity, BackpackProgression, BuildingProgression, ArcadeMachineLocations, HelpWantedLocations, Fishsanity, NumberOfMovementBuffs, \
-    BundleRandomization, BundlePrice, FestivalLocations, FriendsanityHeartSize, ExcludeGingerIsland, TrapItems, Goal, Mods
+    ElevatorProgression, Museumsanity, BackpackProgression, BuildingProgression, ArcadeMachineLocations, QuestLocations, Fishsanity, NumberOfMovementBuffs, \
+    BundleRandomization, BundlePrice, FestivalLocations, FriendsanityHeartSize, ExcludeGingerIsland, TrapItems, Goal, Mods, Monstersanity, Shipsanity, \
+    Cooksanity, Chefsanity, Craftsanity
 
 
-class SVTestCase(unittest.TestCase):
-    player: ClassVar[int] = 1
-    """Set to False to not skip some 'extra' tests"""
-    skip_extra_tests: bool = True
-    """Set to False to run tests that take long"""
-    skip_long_tests: bool = True
+@cache_argsless
+def disable_5_x_x_options():
+    return {
+        Monstersanity.internal_name: Monstersanity.option_none,
+        Shipsanity.internal_name: Shipsanity.option_none,
+        Cooksanity.internal_name: Cooksanity.option_none,
+        Chefsanity.internal_name: Chefsanity.option_none,
+        Craftsanity.internal_name: Craftsanity.option_none
+    }
 
 
-class SVTestBase(WorldTestBase, SVTestCase):
-    game = "Stardew Valley"
-    world: StardewValleyWorld
+@cache_argsless
+def default_4_x_x_options():
```

Comment:
Why do we still have tests for 4.x.x? Do we have plans for cleaning those?

Reply:
Based on a conversation with the author of these presets, @Jouramie , they are left here for performance comparisons and can still be useful, so we should keep them

Reply:
k